README.md

```diff
@@ -19,16 +19,14 @@ This gem adds a few Rake tasks to generate Ruby Interface (RBI) files for dynami
 
 ```
 # -- Gemfile --
-
+# In the default group (same as sorbet-runtime) because of "Features Provided at Runtime" below
```

Comment:
I've heard some people don't use `sorbet-runtime` in the production environment, so I think this part may actually be misleading. Maybe we should say "the group including production environment"